Multisig/UI/Settings/OwnerKeyManagement/ImportOwnerKey/ImportKeyFlow.swift

```diff
@@ -19,49 +19,49 @@ import UIKit
 /// 3. Enter Name (in superclass)
 /// 4. Create Passcode (in superclass)
 class ImportKeyFlow: AddKeyFlow {
-    var keySource: KeySource?
     var flowFactory: ImportKeyFlowFactory {
         factory as! ImportKeyFlowFactory
     }
 
-    enum KeySource {
-        case privateKey
-        case seed
+    var parameters: ImportKeyParameters? {
+        keyParameters as? ImportKeyParameters
     }
 
     init(completion: @escaping (Bool) -> Void) {
-        super.init(keyType: .deviceImported, factory: ImportKeyFlowFactory(), completion: completion)
+        super.init(factory: ImportKeyFlowFactory(), completion: completion)
     }
 
     override func didIntro() {
         enterSecret()
     }
 
     func enterSecret() {
-        keySource = nil
-        privateKey = nil
         let enterVC = flowFactory.enterSecret { [unowned self] privateKey in
-            keySource = .privateKey
-            didGetKey(privateKey: privateKey)
+            keyParameters = ImportKeyParameters(address: privateKey.address,
+                                                name: nil,
+                                                source: .privateKey,
+                                                privateKey: privateKey)
+            didGetKey()
         } completionSeed: { [unowned self] seedNode in
-            keySource = .seed
             pickAccount(seedNode)
         }
         show(enterVC)
     }
 
     func pickAccount(_ node: HDNode) {
         let pickerVC = flowFactory.derivedAccountPicker(node: node) { [unowned self] privateKey in
-            didGetKey(privateKey: privateKey)
+            keyParameters = ImportKeyParameters(address: privateKey.address,
+                                                name: nil,
+                                                source: .seed,
+                                                privateKey: privateKey)
+            didGetKey()
         }
         show(pickerVC)
     }
 
     override func doImport() -> Bool {
-        assert(privateKey != nil)
-        assert(keyName != nil)
-        let success = OwnerKeyController.importKey(privateKey!, name: keyName!, isDrivedFromSeedPhrase: keySource == .seed)
-        return success
+        assert(parameters?.name != nil)
+        return OwnerKeyController.importKey(parameters!.privateKey!, name: parameters!.name!, isDrivedFromSeedPhrase: parameters!.source == .seed)
```

Comment:
previous pattern of `let success = ... ; return success` allows to see the return value when debugging

Reply:
is there a way to see it with the debugger but without such a variable? 🤔

Reply:
There is! https://stackoverflow.com/questions/20902019/inspect-the-return-value-of-a-function-in-lldb